### DISCUSSION TITLE
How to use it?

### DISCUSSION THREAD

**Discussion by AlexxB31** [runew0lf/RuinedFooocus#43] [Q&A] [answered]
Is there a video tutorial how to use it?
Is this just a separate software or is there also an integration for automatic 1111?  

Thanks!

**Comment by runew0lf** [comment]
nope sorry my dude, nobody's done a video on how to use it. Its separate software, not an extension to auto1111

**Comment by barepixels** [comment, marked as answer]
@AlexxB31 There are few Fooocus videos on YouTube.  While they are not specifically for RuinedFooocus one can learn the basics of Fooocus
https://www.youtube.com/results?search_query=fooocus

**Comment by Sairampv98** [comment]
I found this video which explains all the features of ruined focus check it out []()https://www.youtube.com/watch?v=7ncu4f-PFGI

**Comment by Sairampv98** [comment]
I found this video which explains all the features of ruined focus check it out []()https://www.youtube.com/watch?v=7ncu4f-PFGI

**Comment by kengun2077** [comment]
RuinedFooocus indepth guide & detailed usage

https://www.patreon.com/posts/ruinedfooocus-94865185?utm_medium=clipboard_copy&utm_source=copyLink&utm_campaign=postshare_fan&utm_content=web_share
![2](https://github.com/user-attachments/assets/10ea03e6-ce3c-45a5-9cf4-675408b26b09)
